README.md

```diff
@@ -13,15 +13,30 @@ This version (v3) is a brand new implementation in Python 3, supports running on
 
 ### Install Python 3
 
-Install [Python 3](https://www.python.org/) from Linux distribution’s package repositories, or [python.org](https://www.python.org/)
+Install [Python 3.8](https://www.python.org/) from Linux distribution’s package
+repositories, or [python.org](https://www.python.org/). We also need `gcc` and
+the `libpython` headers for the `usjon` package.
+
+On Ubuntu 18.04 and 20.04:
+
+```bash
+sudo apt install git python3.8 libpython3.8-dev python3-distutils python3-apt gcc
```

Comment:
Might be useful to install `build-essentials` instead, as that will also install the libc headers that are necessary to build stuff.

Reply:
Hm, I tested this on a fresh WSL image, I didn't need all of build-essentials to get this working, but only gcc. Normally I'd agree with you though 😂 To be honest, I'd rather avoid needing this at all, but that means replacing `dataclasses-json` (I think) since that's the top level package causing this dependency.

Reply:
I saw that this was tested with a fresh install just after I made the comment :D

(And, yeah, this library seems very nice and plays well with the type annotations.)